index.html

```diff
@@ -102,9 +102,8 @@ <h3>YouTube Video</h3>
                 </p>
                 <!--Found me!-->
                 <picture>
-                    <source type="image/svg" srcset="images/sc-logo.svg">
-                    <source type="image/png" srcset="images/sc-logo.png">
-                    <img src="images/sc-logo.png" srcset="images/sc-logo.svg 70w" alt="Sault College Logo" sizes="(max-width:105px) 105px, (max-width:140px) 140px">
+                    <source type="image/svg+xml" srcset="images/sc-logo.svg">
+                    <img src="images/sc-logo.png" width="70" srcset="images/sc-logo.png, images/sc-logo@1.5x.png 1.5x, images/sc-logo@2x.png 2x" alt="Sault College Logo" >
```

Comment:
You don't need sizes for resolution switch, just a srcset indicating which images correspond to which screen density
Missing width